### DISCUSSION TITLE
Edit USD materials ?

### DISCUSSION THREAD

**Discussion by DavidSpeicher** [Autodesk/maya-usd#2018] [Q&A]
Hello,  
How is it possible to edit Materials or assign materials to USD scenes?

**Comment by kxl-adsk** [comment]
Hi @DavidSpeicher,

No, at this point there is no direct material authoring workflows in MayaUSD. We are aligning internally with another project to deliver a unified experience.

At this point, I can't tell much more.
KxL

**Comment by yolov** [comment]
Hey, hope you don't mind me jumping in. @kxl-adsk does it mean, though, that one can import ("consume) the material data as maya data (there was an option I saw somewhere in 0.15 onwards) - I assume that this should recreate the nodes as a maya shading graph so the user can edit them, or am I wrong ?

**Reply by kxl-adsk** [reply]
Hi @yolov - just letting you know that I no longer work at Autodesk. One of the team members will for sure comment.

From my side, yes, if you have loseless roundtrip of your materials, then you can edit materials in maya's data model.

**Reply by mdubrois** [reply]
@kxl-adsk Sad news for the Maya USD plugin.

**Reply by kxl-adsk** [reply]
There is an amazing team behind this plug-in. My departure won’t change that.

**Reply by williamkrick** [reply]
@yolov as Krystian said you can import the USD into Maya, modify the materials and export them back to USD. One thing to keep in mind is the round trip workflow may lose USD composition information, depending on how your USD stage is built up.

**Comment by yolov** [comment]
Thanks, @kxl-adsk Krystian, that's mighty helpful. Bummer on the job change, you've been very helpful here. Best of luck!

Alex
